test/client-unit/services/PasswordMeterService.spec.js

```diff
@@ -1,94 +1,89 @@
+    expect(PasswordMeterService.validate).to.be.a('function');
   });
 
-  it('#validate() returns true for a medium strength password', () => {
-    const validation = PasswordMeterService.validate(MEDIUM_PASSWORD);
-    expect(validation).to.equal(true);
+  it('#counter() should return -1 for no password', () => {
+    const count = PasswordMeterService.counter();
+    expect(count).to.equal(-1);
   });
 
-  it('#validate() returns false for a weak password', () => {
-    const validation = PasswordMeterService.validate(WEAK_PASSWORD);
-    expect(validation).to.equal(false);
+  it('#validate() should return false for no password', () => {
+    const validate = PasswordMeterService.validate();
+    expect(validate).to.equal(false);
   });
 
-  it('#validate() returns false for an empty password', () => {
-    const validation = PasswordMeterService.validate(EMPTY_PASSWORD);
-    expect(validation).to.equal(false);
+  it('#counter() should return 0 for a weak password', () => {
+    const count = PasswordMeterService.counter(WEAK_PASSWORD);
+    expect(count).to.equal(0);
   });
 
-  it('#validate() returns true for a weak password if password validation is off in enterprise settings', () => {
-    Session.enterprise.settings.enable_password_validation = false;
-    const validation = PasswordMeterService.validate(WEAK_PASSWORD);
-    expect(validation).to.equal(true);
+  it('#validate() should return false for a weak password', () => {
+    const validate = PasswordMeterService.validate(WEAK_PASSWORD);
+    expect(validate).to.equal(false);
   });
 
-  it('#validate() returns true for an empty password if password validation is off in enterprise settings', () => {
-    Session.enterprise.settings.enable_password_validation = false;
-    const validation = PasswordMeterService.validate(EMPTY_PASSWORD);
-    expect(validation).to.equal(true);
+  it('#counter() should return 0 for a medium password', () => {
+    const count = PasswordMeterService.counter(MEDIUM_PASSWORD);
+    expect(count).to.equal(3);
   });
 
-  it('#counter() returns 4 for a strong password', () => {
-    const validation = PasswordMeterService.counter(STRONG_PASSWORD);
-    expect(validation).to.equal(4);
+  it('#validate() should return false for a medium password', () => {
+    const validate = PasswordMeterService.validate(MEDIUM_PASSWORD);
+    expect(validate).to.equal(true);
   });
 
-  it('#counter() returns 3 for a medium-strength password', () => {
-    const validation = PasswordMeterService.counter(MEDIUM_PASSWORD);
-    expect(validation).to.equal(3);
+  it('#counter() should return 0 for a strong password', () => {
+    const count = PasswordMeterService.counter(STRONG_PASSWORD);
+    expect(count).to.equal(4);
   });
 
-  it('#counter() returns 0 for a weak', () => {
-    const validation = PasswordMeterService.counter(WEAK_PASSWORD);
-    expect(validation).to.equal(0);
+  it('#validate() should return false for a strong password', () => {
```

Comment:
String password are what we are expecting the user to enter, go ahead and change this test title to "should return true ...."